server/container/guice/mailbox/src/main/java/org/apache/james/modules/mailbox/DefaultQuotaModule.java

```diff
@@ -38,7 +38,7 @@ protected void configure() {
         bind(DefaultUserQuotaRootResolver.class).in(Scopes.SINGLETON);
         bind(NoMaxQuotaManager.class).in(Scopes.SINGLETON);
 
-        bind(MaxQuotaManager.class).to(NoMaxQuotaManager.class);
+        bind(MaxQuotaManager.class).toInstance(new NoMaxQuotaManager());
```

Comment:
Previous version was better IMO (or `bind(NoMaxQuotaManager.class).in(Scopes.SINGLETON);` becomes useless)

Reply:
Well but Spring rejected inject on constructors without parameters.... What should we do? Patch Spring?